README.md

```diff
@@ -0,0 +1,11 @@
+<h1>Rails Associations</h1>
+
+This collaborative project is focused on Assosiations, as a part of an assignment for microverse using sessions, associations between users, events, and attendances;  where users can create  events, and register himself to different events created by any other user.
+
+
+<h1>Authors:</h1>
+
+<a href="https://github.com/AlejoCode">Daniel Alejandro Salgado</a>
+ --- <a href="https://github.com/gonjavi/">Gonzalo Javier Mancilla</a> --- 
+ 
```

Comment:
Add a more descriptive readme.

Details like assignment link (from Odin), screenshot, how to set up the app in development and what was learned during the project can make the project more descriptive.